typescript/infra/src/utils/safe.ts

```diff
@@ -1,16 +1,23 @@
-import SafeApiKit from '@safe-global/api-kit';
-import Safe, { EthersAdapter } from '@safe-global/protocol-kit';
+import _SafeApiKit from '@safe-global/api-kit';
+import _Safe, { EthersAdapter } from '@safe-global/protocol-kit';
 import { ethers } from 'ethers';
 
-import { ChainName, MultiProvider, chainMetadata } from '@hyperlane-xyz/sdk';
+import { ChainName, MultiProvider } from '@hyperlane-xyz/sdk';
+
+import { getChain } from '../../config/registry.js';
+
+// Due to https://github.com/safe-global/safe-core-sdk/issues/419
+// See also https://github.com/safe-global/safe-core-sdk/issues/514
+const SafeApiKit = _SafeApiKit.default;
+const Safe = _Safe.default;
```

Comment:
Since these are unused, could we instead update the imports to
```
import SafeApiKit from '@safe-global/api-kit';
import Safe, { EthersAdapter } from '@safe-global/protocol-kit';
```
and use `SafeApiKit.default` & `Safe.default` below?

Reply:
I'll give that a try! If typescript accepts it then yes let's do that